pages/troubleshooting/faq.md

```diff
@@ -68,25 +68,18 @@ Yes! All resin.io containers run in privileged mode, which means you can access
 Sometime you may notice the date/time on the device is incorrect, usually via logs.
 
 There seems to be some flakiness with NTP, in theory, it ought to update on connection to the internet (via connman), and then every 2 hours, but appears to fail to do so sometimes altogether.
-A potential cause is the NTP port (123 UDP) being blocked on the network the device belongs to, if this is not the case, the send us a message on support@resin.io
+A potential cause is the NTP port (123 UDP) being blocked on the network the device belongs to, if this is not the case, the send us a message on the [forums][forums].
 
 ##### Can I set a static IP address for my device?
 Yes! its actually pretty easy, have a look at the [ethernet network setup](/deployment/network/#set-static-ip) section of our documentation. In general most network configurations can be achieved by changing the [Connman](http://en.wikipedia.org/wiki/ConnMan) configuration file.
 
 ##### Why can't I SSH into or run code in the HostOS?
```

Comment:
Should we change the title of the FAQ to something like "Why can't I SSH into or run code in older versions of the hostOS?"